core/src/main/scala/zio/logging/Logging.scala

```diff
@@ -1,21 +1,72 @@
 package zio.logging
 
-import zio.{ Cause, ZIO }
+import zio.clock.{ currentDateTime, Clock }
+import zio.console.{ putStrLn, Console }
+import zio.logging.Logger.classNameForLambda
+import zio.{ UIO, URIO, ZIO }
+
+trait Logging[-R] {
+
+  def logger: Logger[R]
 
-trait Logging[Message] {
-  def logging: Logging.Service[Any, Message]
 }
 
 object Logging {
 
-  trait Service[-R, Message] {
+  def make[R](logger: (LogContext, => String) => URIO[R, Unit]) =
```

Comment:
I'd add the return type here because it's useful documentation on top-level functions.

Reply:
of course